src/main/bin/ccm-initialise

```diff
@@ -128,9 +127,9 @@ MakeCacheRoot($this_app, $cache_root, $dopts);
 
 # make global lock file (mask and GID take care of the permissions)
 $this_app->verbose("Creating lockfile");
-open TMP, ">$cache_root/global.lock";
-print TMP "no\n";
-close TMP;
+open my $TMP, ">$cache_root/global.lock";
```

Comment:
I am surprised critic lets this 2 element open() pass? Is it really OK to ignore the return codes of open and close here?